src/content/en/feedback/_index.yaml

```diff
@@ -0,0 +1,121 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
+project_path: /web/_project.yaml
+book_path: /web/_book.yaml
+landing_page:
+  custom_css_path: /web/styles/wf-root.css
+
+  rows:
+
+  - heading: Developer Feedback
+    classname: devsite-landing-row-67
+    items:
+    - description: >
+        <p>
+          We’re committed to making your site work across platforms
+          and browsers so you can be successful. We want to make it
+          easy to find and report bugs in any browser, as well as
+          help you find resources while you’re building your site.
+         </p>
+         <p>&nbsp;</p>
+      image_path: images/hero-2x.png
+      buttons:
+      - label: Search bugs in all web browsers
+        path: "#search-bugs-in-all-web-browsers"
+        classname: button button-primary
+      - label: Reporting bugs
+        path: "#reporting-bugs"
+        classname: button button-primary
+      - label: Discussing solutions
+        path: "#discussing-solutions"
+        classname: button button-primary
+
+  - items:
+    - custom_html: <hr>
+
+  - heading: Search bugs in all web browsers
+    items:
+    - image_path: images/content-search-for-bugs-responsive-2x.png
+    - description: >
+          Here you can search through all major browser bug trackers to find a bug you're looking for.
+          <p>&nbsp;</p>
+          <iframe src="https://browser-issue-tracker-search.appspot.com/?embed=1" width="100%" height="250" scrolling="no"></iframe>
+
+  - items:
+    - custom_html: <hr>
+
+  - heading: Reporting bugs
+    classname: devsite-landing-row-no-image-background
+    items:
+    - heading: Cross-browser testing
+      description: >
+        We recommend that you test in multiple web browsers on
+        multiple platforms. Real machines and devices are best,
+        but another great option is using BrowserStack.
+      image_path: images/content-node-responsive-2x.png
+      buttons:
+      - label: BrowserStack
+        path: https://www.browserstack.com/
+
+    - heading: How to file a good bug
+      description: >
+        When filing bugs, the more information the better.
+        If you have a reproducible test with a limited scope
+        there's a better chance your bug will be solved.
+      image_path: images/content-file-responsive-2x.png
+      buttons:
+      - label: Learn more
+        path: https://crbug.com
+      - label: How to create a minimized reproducable test case
+        path: https://crbug.com
+        classname: wf-small-link
+      - label: How to file a good bug
+        path: /web/feedback/file-a-bug/
+        classname: wf-small-link
+      - label: JSFiddle
+        path: https://jsfiddle.net/
+        classname: wf-small-link
+      - label: Codepen
+        path: https://codepen.io/
+        classname: wf-small-link
+
+    - heading: Reporting bugs
+      description: >
+        If you encounter browser issues, adding support to
```

Comment:
Dru's wording here, but the idea is that if you add information to existing bugs or report new ones helps the web platform as a whole.